docs/setup/_setup_main.md

```diff
@@ -141,6 +141,15 @@ flutterfire configure
     flutter run
     ```
 
+If you would rather use a demo project, you can start the [Firebase Emulator](https://firebase.google.com/docs/emulator-suite) and
+in your `lib/main.dart` file initialize Firebase using
```

Comment:
I wonder if we should also mention that the project id ought to start with `demo-` to work 🤔 .

Reply:
I've added :thumbsup: